src/Neo/Network/P2P/Payloads/Conflicts.cs

```diff
@@ -15,7 +15,7 @@ public class Conflicts : TransactionAttribute
 
         public override TransactionAttributeType Type => TransactionAttributeType.Conflicts;
 
-        public override bool AllowMultiple => true;
+        public override int AllowedCount => 4;
```

Comment:
Unfortunately, this won't solve the issue. We may have a lot of transactions with the same single conflicting attribute, they will be stored and the problem will repeat. What we need is to check the number of *already stored* conflicting hashes in the `ConflictingSigners` field and if new mempooled transaction contains  yet another signer that should be added to this list (and exceed the limit), then we need to reject this transaction.

Reply:
@AnnaShaleva Please, check the last change